.changelog/570.bugfix.md

```diff
@@ -0,0 +1 @@
+Add token overview page and dashboard component
```

Comment:
This should be improved

Reply:
This is the same so it can be deduplicated by towncrier (when a new feature and subsequent improvements are within one release)

Reply:
Ahh, I see. Btw there is also a `trivial` filename format, which is dropped by towncrier..

Reply:
yup. I'll change to that, it's just less confusing